lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -124,6 +124,15 @@ def persp_transformation(zfront, zback):
                      [0,0,-1,0]
                      ])
 
+def ortho_transformation(zfront, zback):
+    a = -(zfront + zback)
+    b = -(zfront - zback)
+    return np.array([[2,0,0,0],
+                     [0,2,0,0],
+                     [0,0,-2,0],
+                     [0,0,a,b]
+                     ])
```

Comment:
ah, and if I transpose that, it then matches up. Got it. Well, just include some notes about it in comments for some future poor soul.